src/Chainweb/Pact/Types.hs

```diff
@@ -429,7 +429,7 @@ defaultPactServiceConfig = PactServiceConfig
 -- is initialized from the @_configBlockGasLimit@ value of @ChainwebConfiguration@.
 --
 defaultBlockGasLimit :: GasLimit
-defaultBlockGasLimit = 10000
+defaultBlockGasLimit = 20000
```

Comment:
Tests which used testing mempools in `PactMultiChainTest` stopped working because those mempools are willing to provide blocks of transactions that exceed the block gas limit, which was now being enforced.